server/requirements.txt

```diff
@@ -20,7 +20,11 @@ Jinja2==2.10.3
 Mako==1.1.0
 MarkupSafe==1.1.1
 oauth2client==4.1.3
+<<<<<<< HEAD
```

Comment:
Probably want to remove this file altogether, but definitely fix the merge conflicts.